ITGlueAPI/ITGlueAPI.psd1

```diff
@@ -18,7 +18,7 @@
     # -- MINOR version when you add functionality in a backwards-compatible manner, and
     # -- PATCH version when you make backwards-compatible bug fixes.
 
-    ModuleVersion = '2.1.0'
+    ModuleVersion = '2.2.0'
```

Comment:
@davidhaymond made me aware via #148 that my own instructions from a few years back asked for a ModuleVersion bump, so that's entirely on me 🙃 

I went ahead and committed the change to revert the bump and will merge this PR once the other notes about `updated_at` and -`updated_at` are addressed.

Thanks!